python/ray/autoscaler/_private/util.py

```diff
@@ -78,11 +78,6 @@ def validate_config(config: Dict[str, Any]) -> None:
     try:
         import jsonschema
     except (ModuleNotFoundError, ImportError) as e:
-        logger.warning(
```

Comment:
can you leave a note why no warning is needed here in the comments?

Reply:
yep, done!